.github/workflows/pr-checks.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     
     # Install .NET Core
     - name: Install .NET ${{ matrix.dotnet-version }}
-      if: ${{ matrix.os == 'ubuntu-latest' }}
+      if: ${{ matrix.os == 'ubuntu-22.04' }}
```

Comment:
Let's keep this at `latest`

Reply:
Ok.  I missed that through the merge process.